lib/Migration/Version3600Date20240220134813.php

```diff
@@ -61,4 +68,17 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		return $schema;
 	}
 
+	public function postSchemaChange(IOutput $output, \Closure $schemaClosure, array $options) {
+		// Let's buffer this a bit
+		$aMinuteAgo = time() - 60;
+		/** @var QueryBuilder $query */
+		$query = $this->connection->getQueryBuilder();
+		$query->update('mail_local_messages')
+			->set('status', $query->createNamedParameter(LocalMessage::STATUS_IMAP_SENT_MAILBOX_FAIL, IQueryBuilder::PARAM_INT))
```

Comment:
this will explode for we updates if the LocalMessage class has been loaded before the code is replaced and the update is run. Inline the const.